### DISCUSSION TITLE
Is there a way to turn collaboration on for the lexical playground? I don't see a setting for it

### DISCUSSION THREAD

**Discussion by 6peterlu** [facebook/lexical#3259] [Q&A] [answered]
<img width="218" alt="image" src="https://user-images.githubusercontent.com/8334251/197936827-fe35460a-9ea7-4fbc-8b87-232e9ab46c12.png">

I'm confused because I see code for the CollaborationPlugin in the source: https://github.com/facebook/lexical/blob/main/packages/lexical-playground/src/Editor.tsx

**Comment by trueadm** [comment, marked as answer]
You’ll need to checkout the playground locally. The the toggle will show up for you :) we disabled it on the website as we don’t have a public server running nor do we want to run one that we need to monitor.
